include/mxnet/tensor_blob.h

```diff
@@ -104,6 +104,14 @@ class TBlob {
       : dptr_(dptr), shape_(shape), type_flag_(type_flag) {
     SetDLTensor(dev_mask, dev_id);
   }
+  /*!
+   * \brief constructor that construct TBlob from DLTensor
+   * \param DLTensor Object
+   */
+  explicit TBlob(const DLTensor &dltensor) : dptr_(dltensor.data),
```

Comment:
Need to add compactness check

Reply:
Specifically, TBlob only support compact tensors, need to check strides == null or the strides reflect a compact setting

Reply:
Thanks. I will move the strides check from ndarray.cpp to tensor_blob.h.